syntax = "proto3";

package pedro.pb;

message CounterRequest {
    int32 delta = 1;
}

message CounterResponse {
    int64 value = 1;
}